src/store.rs

```diff
@@ -0,0 +1,172 @@
+use std::path::{Path, PathBuf};
+
+use rocksdb::{ColumnFamilyDescriptor, DBWithThreadMode, MultiThreaded};
+
+use crate::{block::store::BlockStore, state::ledger::store::LedgerStore};
+
+#[derive(Debug)]
+pub struct IndexerStore {
+    db_path: PathBuf,
+    database: DBWithThreadMode<MultiThreaded>,
```

Comment:
Are we having multiple producers to this database?

Reply:
One producer, multiple column families!